.github/workflows/linux.yml

```diff
@@ -26,6 +26,15 @@ jobs:
         CONFIGURATION=${{matrix.configuration}}
         ARCH=${{matrix.platform}}
         source ./github_build.sh
+    - uses: actions/upload-artifact@v3
```

Comment:
Are you sure you want to do uploads on each CI build? I mean it's handy - but we have run github space a while back requiring deletions. Guess we can see if that happens again.

Reply:
Yeah, this allows me to release an extension without creating a Slang release first. The default policy is that it will stay for 90 days. Perhaps we can change that to something like 1 day and see if it works.